mmdet/core/anchor/anchor_generator.py

```diff
@@ -1,54 +1,141 @@
+import numpy as np
 import torch
 
+from .registry import ANCHOR_GENERATORS
 
+
+@ANCHOR_GENERATORS.register_module
 class AnchorGenerator(object):
-    """
+    """Standard anchor generator for 2D anchor-based detectors
+
+    Args:
+        strides (list[int]): Strides of anchors in multiple feture levels.
+        base_size (list[int]): The basic sizes of anchors in multiple levels.
+        scales (list[int]): Anchor scales for anchors in a single level.
+        ratios (list[float]): The list of ratios between the height and width
+            of anchors in a single level.
```

Comment:
The argument order in the docstring does not match the order in `__init__` method.